flink-ml-lib/src/test/java/org/apache/flink/ml/feature/UnivariateFeatureSelectorTest.java

```diff
@@ -0,0 +1,783 @@
+        Configuration config = new Configuration();
+        config.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, true);
+        env = StreamExecutionEnvironment.getExecutionEnvironment(config);
+        env.setParallelism(4);
+        env.enableCheckpointing(100);
+        env.setRestartStrategy(RestartStrategies.noRestart());
+        env.getConfig().enableObjectReuse();
+        tEnv = StreamTableEnvironment.create(env);
+
+        selectorWithChiSqTest =
+                new UnivariateFeatureSelector()
+                        .setFeatureType("categorical")
+                        .setLabelType("categorical");
+        selectorWithANOVATest =
+                new UnivariateFeatureSelector()
+                        .setFeatureType("continuous")
+                        .setLabelType("categorical");
+        selectorWithFValueTest =
+                new UnivariateFeatureSelector()
+                        .setFeatureType("continuous")
+                        .setLabelType("continuous");
+        inputChiSqTable =
+                tEnv.fromDataStream(
+                                env.fromCollection(
+                                        INPUT_CHISQ_DATA,
+                                        Types.ROW(Types.DOUBLE, VectorTypeInfo.INSTANCE)))
+                        .as("label", "features");
+        inputANOVATable =
+                tEnv.fromDataStream(
+                                env.fromCollection(
+                                        INPUT_ANOVA_DATA,
+                                        Types.ROW(Types.INT, VectorTypeInfo.INSTANCE)))
+                        .as("label", "features");
+        inputFValueTable =
+                tEnv.fromDataStream(
+                                env.fromCollection(
+                                        INPUT_FVALUE_DATA,
+                                        Types.ROW(Types.DOUBLE, VectorTypeInfo.INSTANCE)))
+                        .as("label", "features");
+    }
+
+    private void transformAndVerify(
+            UnivariateFeatureSelector selector, Table table, int... expectedIndices)
+            throws Exception {
+        UnivariateFeatureSelectorModel model = selector.fit(table);
+        Table output = model.transform(table)[0];
+        verifyOutputResult(output, expectedIndices);
+    }
+
+    private void verifyOutputResult(Table table, int... expectedIndices) throws Exception {
+        StreamTableEnvironment tEnv =
+                (StreamTableEnvironment) ((TableImpl) table).getTableEnvironment();
+        CloseableIterator<Row> rowIterator = tEnv.toDataStream(table).executeAndCollect();
+        while (rowIterator.hasNext()) {
+            Row row = rowIterator.next();
+            for (int i = 0; i < expectedIndices.length; i++) {
+                assertEquals(
+                        ((Vector) row.getField("features")).get(expectedIndices[i]),
+                        ((Vector) row.getField("output")).get(i),
```

Comment:
Should we also verify that the given `table` only contains the selected features?